.github/workflows/pull_request.yml

```diff
@@ -30,6 +30,12 @@ jobs:
     - name: run unit tests
       run: ./mvnw clean test -pl \!samples -am -Dmaven.javadoc.skip=true -T 1C
 
+    - name: Run codacy-coverage-reporter
+      uses: codacy/codacy-coverage-reporter-action@0.2.0
+      with:
+        project-token: ${{ secrets.CODACY_PROJECT_TOKEN }}
+        coverage-reports: ./coverage/target/site/jacoco-aggregate/jacoco.xml
```

Comment:
let's use `./target/site/jacoco-aggregate/jacoco.xml`

Reply:
This path is where the `jacoco.xml` is generated currently and it's under `coverage` module. Do we want to bring it to project level and remove the `coverage` module?